django/db/models/fields/__init__.py

```diff
@@ -82,6 +82,7 @@ class NOT_PROVIDED:
 
 # The values to use for "blank" in SelectFields. Will be appended to the start
 # of most "choices" lists.
+# ToDo: write deprecation notice per Trac issue #35870.
```

Comment:
Do we want to keep the original `BLANK_CHOICE_DASH` in favor of a deprecation notice?

Reply:
We need to discuss exactly what to do here. Folks in the ecosystem are using this: 

https://github.com/search?q=BLANK_CHOICE_DASH+language%3APython&type=code&l=Python

TBH I have to think about it a bit to know exactly what to say. It's delicate.